src/Image/Cache.php

```diff
@@ -64,116 +57,79 @@ class Cache
      */
     static function resolve_url($url, $protocol, $host, $base_path, Dompdf $dompdf)
     {
-        self::$_dompdf = $dompdf;
+        $options = $dompdf->getOptions();
```

Comment:
Maybe use the opportunity to pass the options directly instead of the Dompdf instance here? Would make the dependency clearer.

Reply:
I was trying to decide between that change and API compatibility and settled on compatibility. This is a new major release, though, so a breaking change would be OK. Not that (m)any people would be calling this directly.